extensions/amp-story/1.0/amp-story-page-attachment-header.css

```diff
@@ -38,14 +38,15 @@
   cursor: pointer !important;
 }
 
-.i-amphtml-story-page-attachment-dark-mode .i-amphtml-story-page-attachment-close-button {
+.i-amphtml-story-page-attachment-theme-dark .i-amphtml-story-page-attachment-close-button {
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 36 36" fill="rgba(255, 255, 255, 0.54)"><path d="M28.5 9.62L26.38 7.5 18 15.88 9.62 7.5 7.5 9.62 15.88 18 7.5 26.38l2.12 2.12L18 20.12l8.38 8.38 2.12-2.12L20.12 18z"/><path d="M0 0h36v36H0z" fill="none"/></svg>') !important;
 }
 
 .i-amphtml-story-page-attachment-title {
   font-family: 'Roboto', sans-serif !important;
   width: calc(100% - 80px) !important;
   padding-right: 40px !important;
+  color: rgba(0, 0, 0, 0.87) !important;
```

Comment:
Was this intentional? 🤔

Reply:
Yep, I broke it sometimes last week, it's actually fixing a bug where the title would be white on white :)